Dockerfile

```diff
@@ -1,21 +1,20 @@
-FROM quay.io/centos/centos:stream9-development
-
 ARG USER_ID=${USER_ID:-1001}
 ARG APP_DIR=${APP_DIR:-/app}
+
+FROM quay.io/centos/centos:stream9-development as build
```

Comment:
Can you look to use stream9-minimal? see https://github.com/ansible/ansible-rulebook/pull/635

Reply:
@Alex-Izquierdo I will try the regular, non minimal, version for the build stage. Minimal will be missing a lot of packages needed in the build phase.

Reply:
https://github.com/ansible/ansible-rulebook/pull/639/commits/9ab6ad4022996a2ece0b06e2069ba5ba7cf10680